query/src/configs/config_meta.rs

```diff
@@ -22,6 +22,15 @@ use common_meta_grpc::MetaGrpcClientConf;
 use serde::Deserialize;
 use serde::Serialize;
 
+// Meta env.
+pub const META_ADDRESS: &str = "META_ADDRESS";
```

Comment:
Hmm, why add these env back?

Reply:
I think not use them anymore since this PR a few days ago:
https://github.com/datafuselabs/databend/pull/4971/files#diff-51b91c219c223ffd4cf157ac0a9f3e44bf2f6ec489dcfbe3682a816d9a337dbe